README.md

```diff
@@ -96,18 +96,18 @@ Recommended practices
 
 ### Queries
 
-- Should be defined in files named as `sql/<dataset>/<table>_<version>/query.sql` e.g.
-  `sql/telemetry_derived/clients_daily_v7/query.sql`
+- Should be defined in files named as `<project>/sql/<dataset>/<table>_<version>/query.sql` e.g.
```

Comment:
I wonder if we can also drop `sql/` at some point (or now?) 🤔 then the structure bigquery-etl would reflect the BigQuery project/dataset/table structure.

Reply:
> Then the bigquery-etl structure would reflect the BigQuery project/dataset/table structure

I like that property, so I'm definitely in favor of dropping `sql` between project and dataset, but I imagined there would still be a top-level `sql` directory that projects are nested under. In that scenario, we would be pulling the existing `mozfun` under `sql` as well. 

I suppose this gets more muddy for `udf` and `udf_js` which create functions in `shared-prod`. I think my ideal situation is that those are pulled underneath `sql/moz-fx-data-shared-prod/` so that the `sql` directory truly maps directly to the project/dataset/table layout in BQ. That likely would require more intensive modification of UDF logic, though. Does that seem doable?

I worry about a proliferation of top-level directories in the repo.